nrfutil/device/batch.ts

```diff
@@ -133,21 +154,34 @@ export class Batch {
         count: number,
         callback: (completedTasks: TaskEnd<unknown>[]) => void
     ) {
+        if (!callback || typeof count === 'undefined') {
+            // handle if not passed: log use case or anything else
+            // same check can be applied to the methods above if it makes sense
+
+            return this;
+        }
```

Comment:
Why add these checks when they should never happen according to the types? If we want to become more liberal on what we accept as arguments then we should also change the types of the parameters but I currently do not see an advantage in it.